### DISCUSSION TITLE
Friends and personal records merged view

### DISCUSSION THREAD

**Discussion by 20Attila03** [ppy/osu#13172] [Ideas]
**Describe the new feature:** 
I would like to see a ranking page called "Friends and you". This tab could show you your friends best scores, your best score in order and after that with a divider line all of your friend's and your own scores also in order.

**Proposal designs of the feature:** 
I don't have a detailed idea of how I would make it look like, but I think the feature description summarizes it perfectly.

**Comment by peppy** [comment]
I don't understand. Are you talking about seeing *all* local scores in the friend leaderboard?

**Comment by 20Attila03** [comment]
Well, yes but in a different tab.

I would like to have a tab with only friend's scores in it and with my scores in it as well.

**Comment by peppy** [comment]
Could this just be a toggle to always show local scores in all scoreboards (including globa/mods as well)?

**Comment by 20Attila03** [comment]
In my opinion, yes. That could be a solution

**Comment by LittleEndu** [comment]
Maybe a toggle similar to (like positioned next to) "Show scores only from selected mods"? (or what ever that toggle was called exactly)

**Comment by 20Attila03** [comment]
It would be cool as it is easy access as well.
